stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/state/MessageMode.kt

```diff
@@ -1,4 +1,4 @@
-package io.getstream.chat.android.compose.state.messages
+package com.getstream.sdk.chat.state
```

Comment:
We really need to fix these packages names in the common module at some point. Perhaps we should already have these in an `io.getstream.*` package even if old code in the common module is in `com.getstream`. At least not to add new code under the old package name.

Reply:
Should the base package name in the common module (for `MessageComposerController`, `MessageMode`, and `MessageAction` classes)  be `io.getstream.chat.android` or `io.getstream.chat.android.common`? cc @filip

Reply:
Yeah that makes sense, we can move all the code to io.getstream.*, I'll update this and then we can review

Reply:
I think `io.getstream.chat.android.ui` or `io.getstream.chat.android.ui.common` would be appropriate for the package name here. The former is simpler, maybe we can go with that.